built_value/pubspec.yaml

```diff
@@ -8,7 +8,7 @@ topics:
  - built-value
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=2.12.0 <4.0.0'
```

Comment:
since `RecordType` is 3.0.0 , will this cause problems for packages that are <3.0.0 ?